lib/records/fhir_dstu2.rb

```diff
@@ -332,6 +353,29 @@ def self.observation(observation, fhir_record, patient, encounter)
         fhir_record.entry << entry
       end
 
+      def self.quality_of_life_observation(observation, fhir_record, patient)
```

Comment:
Yeah. I figured, otherwise, you would just collapse this down to one class. That said, for code where you can, pull it out into the version independent file. For places where you need variation, you could leave the independent code alone or do something like:

```ruby
def self.do_something(thing, fhir_record, patient)
  entry = FHIRVersionIndependentRecord.do_something(thing, fhir_record, patient, FHIR)
  entry.tweak_thing_for_this_version
  fhir_record.entry << entry
end
```

I don't know how you feel about this approach, but it just struck me how similar the two files are.